### PR TITLE
🎨 [UI] Implement authentication page design

### PR DESCRIPTION
**Pull request by Khadeeejah**
Created the UI for the authentication page 
fully responsive on all screens

bugs:
the footer seems to be distorting on every refresh, please check.

waiting for a feedback

ISSUE NO #12

### CODE REVIEW

**Review by MeisterJustice — CHANGES_REQUESTED** [review]
Looks good, but a minor comment. Could you properly format your code please? some of it is kind of difficult to read. 

Also, the top spacings on your implementation doesn't match what's on the design. Please check

Screenshot 2022-06-30 at 22 13 32

**Comment by Khadeeejah** [comment]
hi @MeisterJustice , please review the changes i made

File: content/docs/quick-start/authentication.mdx [hunk 1]
```diff
@@ -9,44 +8,32 @@ the request you're making in the header of request you make.
...
-
     <Alert 
     status="warning" 
     title="Secure your requests"
     className="my-2" 
     content=' Do not commit your secret keys to Git, or use them in client-side code..'/>
 
-<Divider className="max-w-100 mb-10  mt-5" />
-
+<Divider className='max-w-100  mb-12' />
```
Comment (MeisterJustice) [inline content/docs/quick-start/authentication.mdx:17]: Please check if `mb-12` works here.  Still doesn't match the design

**Review by MeisterJustice — CHANGES_REQUESTED** [review]

File: content/docs/quick-start/authentication.mdx (hunk 1 above)
Comment (Khadeeejah) [reply content/docs/quick-start/authentication.mdx:17]: image

Hi this is what it looks like with mb-12

Comment (MeisterJustice) [reply content/docs/quick-start/authentication.mdx:17]: I mean, it's currently `mb-12`, which i don't think works

**Comment by Khadeeejah** [comment]
It seems we are both seeing different things on our local machines. image


That's what I'm currently seeing without making any changes to the margin

**Pull request merged**